src/style.scss

```diff
@@ -40,6 +40,9 @@
 * {
     font-family: 'Nunito Sans', sans-serif;
 }
+.container {
+    padding: 0px !important;
```

Comment:
bootstrap' ten kenar boşluklarını kaldırmak için yaptık

Reply:
minik bi info eger margin, padding vs yi sifirlamak istiyosaniz birim vermenize gerek yok. direkt ` padding: 0 !important` yazabilirsiniz ornegin